Codeinwp/themeisle-sdk
# feat: add neve fse notice on theme-install page #175

### Summary
Will add a promotion notice on the Theme Install Page for Neve FSE if the WP version is higher or equal to 6.2
The link will open the Neve FSE page for now.

### Screenshots
![image](https://github.com/Codeinwp/themeisle-sdk/assets/23024731/0f772946-b5fc-463d-ab58-7484e8eccbd2)

### How to test
1. Use this build of the SDK
2. Have an instance with WP6.2 or higher
3. The Product that is using the SDK has to have an install time older than 4 days.
4. Check that the notice appears inside `wp-admin/theme-install.php?browse=popular`
5. Check that the Learn More will open a new tab with the Neve FSE page from Themeisle
6. Check that it can be dismissed

Closes: #175.
